packages/apps/transfer/src/components/Global/Ace/index.tsx

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line simple-import-sort/imports
+import React, { FC } from 'react';
+import AceEditor from 'react-ace';
+
+import 'ace-builds/src-noconflict/mode-clojure';
+import 'ace-builds/src-noconflict/mode-lisp';
+import 'ace-builds/src-noconflict/theme-monokai';
+import 'ace-builds/src-noconflict/theme-one_dark';
+
+export interface AceEditorProps {
+  code?: string;
+}
+
+const AceViewerComponent: FC<AceEditorProps> = ({ code }) => {
```

Comment:
Question: I haven't started the app so maybe this is irrelevant. Do we expose the option to enable vim-mode, and maybe other useful QoL features?

Reply:
We don't need it to be an editor, just a code viewer, therefore we kept it like this